using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -426,7 +488,9 @@ Retrieve Adobe visitor data as a URL query parameter string for consumption in h
 public static void getUrlVariables(final AdobeCallback<String> callback);
 ```
 
-#### **Example**
+o   *callback* is a block pointer to call with an NSString value that contains the visitor identifiers as a querystring upon completion of the service request.
```

Comment:
- Please move to after the code example.
- Please reword to say:
"*callback* is a block pointer with an NSString value that contains the visitor identifiers as a querystring when the service request is complete.